orchagent/vxlanorch.cpp

```diff
@@ -228,7 +228,8 @@ create_tunnel(
     sai_object_id_t tunnel_encap_id,
     sai_object_id_t tunnel_decap_id,
     sai_ip_address_t *src_ip,
-    sai_object_id_t underlay_rif)
+    sai_object_id_t underlay_rif,
+    sai_uint8_t encap_ttl=0)
```

Comment:
no need to specify the default value here, as already specified in the header file

Reply:
This is ```static``` helper function to create tunnel in SAI, so we need default value here (in header file default value specified for "create tunnel" public APIs)